backend/src/main/java/com/redhat/cloud/notifications/routers/handlers/endpoint/EndpointResourceV2.java

```diff
@@ -39,13 +45,36 @@
 import java.util.Set;
 import java.util.UUID;
 
+import static com.redhat.cloud.notifications.Constants.API_INTEGRATIONS_V_2_0;
 import static com.redhat.cloud.notifications.db.Query.DEFAULT_RESULTS_PER_PAGE;
 import static com.redhat.cloud.notifications.routers.SecurityContextUtil.getOrgId;
 import static com.redhat.cloud.notifications.routers.SecurityContextUtil.getUsername;
 import static jakarta.ws.rs.core.MediaType.APPLICATION_JSON;
 
-@Path(Constants.API_INTEGRATIONS_V_2_0 + "/endpoints")
-public class EndpointResourceV2 extends EndpointResource {
+public class EndpointResourceV2 extends EndpointResourceCommon {
+
+    @Inject
+    BackendConfig backendConfig;
+
+    @Inject
+    CommonMapper commonMapper;
+
+    @Inject
+    EndpointRepository endpointRepository;
+
+    @Inject
+    NotificationRepository notificationRepository;
+
+    @Inject
+    WorkspaceUtils workspaceUtils;
+
+    @Inject
+    KesselInventoryAuthorization kesselInventoryAuthorization;
```

Comment:
`BackendConfig`, `CommonMapper`, `EndpointRepository` and `KesselInventoryAuthorization` are already injected in the parent class and are package-protected which should make them available here. Do we really need to reinject them in this class?